draft-ietf-ppm-dap.md

```diff
@@ -2653,24 +2513,26 @@ reveal additional information about the honest measurements, leading to a
 privacy violation; or the result may have some property that is desirable to the
 adversary ("stats poisoning").
 
-Depending on the deployment and threat model, there are different ways to address
-Sybil attacks, including, though not limited to:
+Depending on the deployment and the specific threat being mitigated, there are
+different ways to address Sybil attacks, such as:
 
-1. Authenticating clients, as described in {{client-auth}};
-1. Removing client-specific metadata on individual reports, such as through the
-   use of anonymizing proxies in the upload flow, as described in {{anon-proxy}}; or
+1. Implementing Client authentication, as described in {{client-auth}}, likely
+   paired with rate-limiting uploads from individual Clients.
+1. Removing Client-specific metadata on individual reports, such as through the
+   use of anonymizing proxies in the upload flow, as described in
+   {{anon-proxy}}.
 1. Applying differential privacy protections, as described in {{dp}}.
```

Comment:
We don't want to over-promise here. Whether/how DP is used to beat Sybil attacks is not straightforward.
```suggestion
1. Differential privacy ({{dp}}) can help mitigate Sybil attacks to some extent.
```